test/location_data_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:one_app/utils/data_service.dart';
+import 'package:one_app/utils/location_data.dart';
+import 'package:one_app/utils/weather_data.dart';
+import './mock_data.dart';
+
+class MockDataService extends Mock implements DataService {}
+
+void main() {
+  late MockDataService mockDataService;
+  late LocationData locationData;
+
+  setUp(() {
+    mockDataService = MockDataService();
+    locationData = LocationData('Tokyo', mockDataService);
+  });
+
+  test(
+    'initial values are correct',
+    () {
```

Comment:
The framework style guide always has these as one-liners, regardless of line length.

```suggestion
  test('initial values are correct', () {
```